.classpath

```diff
@@ -8,103 +8,103 @@
 			<attribute name="module" value="true"/>
 		</attributes>
 	</classpathentry>
-	<classpathentry kind="lib" path="/jars/lib/jars/apache_commons/commons-logging-1.1.1.jar"/>
-	<classpathentry kind="lib" path="/jars/lib/jars/ini4j/ini4j-0.5.2.jar"/>
-	<classpathentry kind="lib" path="/jars/lib/jars/jetty/jetty-all-7.0.0.jar"/>
-	<classpathentry kind="lib" path="/jars/lib/jars/jna/jna-3.4.0.jar"/>
-	<classpathentry kind="lib" path="/jars/lib/jars/servlet/servlet-api-2.5.jar"/>
-	<classpathentry kind="lib" path="/jars/lib/jars/syslog4j/syslog4j-0.9.46.jar"/>
-	<classpathentry kind="lib" path="/jars/lib/jars/apache_commons/commons-collections-3.2.1.jar" sourcepath="/jars/lib/jars/apache_commons/commons-collections-3.2.1-sources.jar">
+	<classpathentry kind="lib" path="/Users/gwg/code/kbase/jars/lib/jars/apache_commons/commons-logging-1.1.1.jar"/>
```

Comment:
This breaks the classpath file for everyone

Reply:
Accidental commit, now removed

Reply:
:+1: